### DISCUSSION TITLE
COMPLAINT TO GITHUB - GitHub Website itself on Android mobiles is horribly buggy!

### DISCUSSION THREAD

**Discussion by Morsmalleo** [community/community#43089] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bugs with the GitHub website on Android Mobile Devices.

### Body

I have no idea where to write this anymore, I've tried to look for a way to contact the developers of GitHub itself but I can't find anything, so I'll post it here! This is about the GitHub website itself, not an application or project!

When I'm visiting any of my repositories on GitHub using an Android mobile, things appear to be fine, however when I want do something like quickly edit my README.md file or for that matter, any file on any of my repositories, the code editor for editing any file from GitHub on your repositories is completely horrid when being used on Android!

This only started happening in the last couple of months ago after GitHub upgraded their website more.
- Highlighting text is a pain in the ass because it either skips what you're highlighting and highlights a completely different section of code.

- Removing empty lines from when editing files in general is horrid as it everytime you press backspace at the beginning of an empty line in order to remove it, all it does is just close the on screen keyboard with removing the empty line, you actually have to enter a letter from the keyboard and snap backspace twice very fast to achieve removing empty lines when editing files on GitHub with Android.

- Tapping on any place to edit any section of file on GitHub using Android is buggy as well because it just throws you right back to your previous position of editing.

I didn't know where else to post this, so hopefully the GitHub Devs catch it, hopefully someone else has experienced what I'm experiencing as well.

For the record, everything works fine on a PC, it's just buggy asf on Android, when it never used to be.

**Comment by radar-blip** [comment, marked as answer]
You can try filing a [Bug-Report](https://support.github.com/contact/bug-report) and explain the same things that you explained here. 
It cant hurt, might even get some things done. Just give them all the details you can think of.
Good luck!

**Reply by Morsmalleo** [reply]
Thank you man, I don't know how I missed this one, but thank you.

**Reply by radar-blip** [reply]
No problem. Hopefully you get some good results. 👍🏻
